doc/source/contributing/how_to_contribute.rst

```diff
@@ -94,10 +102,10 @@ environment (from the home level of your local copy of the icepyx repo) with
 
     mamba env create --name icepyx-env --channel conda-forge -f requirements-dev.txt -f requirements.txt
```

Comment:
Any idea how easy it is to add the interactive "copy" button to the rendered version of this? Definitely not something we should hold up this PR for, but this caught my eye during review as a spot that would be nice to have.

Reply:
I think that's a pretty easy change with a plugin: https://sphinx-copybutton.readthedocs.io/en/latest/

Reply:
#587